spec.bs

```diff
@@ -1563,3 +1598,133 @@ specification adds another value for fenced frames:
 
 : "`fencedframe`"
 :: This [=navigable=] is displaying a <{fencedframe}>'s content
+
+<h3 id=permissions-policy-changes>Permissions Policies</h3>
+Permissions are granted to fenced frames through the FencedFrameConfig object, which defines the
+required permissions for the fenced frame to load. The fenced frame is only allowed to navigate if
+all of the required permissions are present in the context loading the fenced frame. When a fenced
+frame loads, it is assumed that the permissions checks passed, and the fenced frame will load with
+the list of permissions that were specified in the config object.
+
+Modify the [=Create a Permissions Policy for a navigable=] algorithm:
+
+<div algorithm=create-permissions-policy>
+  Modify the definition of [=Create a Permissions Policy for a navigable=] to read:
+
+  Given null or an element (|container|), an
+  [=origin=] (<var ignore>origin</var>), and null or a
+  list of [=features=] (|required features|), this algorithm returns a new
+  [=Document/permissions policy=].
+
+  Rewrite step 1 of the algorithm to read:
+
+  1. Assert: if not null, |container| is either a [=navigable container=] or a
+    [=fenced navigable container=].
+
+  Add a new step after step 1 that reads:
+
+  2. Assert: if |container| is not a [=fenced navigable container=], |required features| is null.
+
+  Rewrite step 4 to read:
+
+  4. If |container| is a [=fenced navigable container=], then for each |feature| supported,
+
+      1. If |feature| exists in |required features|, set |inherited policy|[|feature|] to "`Enabled`".
+
+        Otherwise, set |inherited policy|[|feature|] to "`Disabled`".
+
+    Otherwise, for each |feature| supported,
+
+      1. Let |isInherited| be the result of running [=Define an inherited policy for feature in
+        container at origin=] on |feature|, |container| and <var ignore>origin</var>.
+
+      2. Set |inherited policy|[|feature|] to |isInherited|.
+</div>
+
+<div algorithm=allow-attribute-fenced-frame>
+  Modify the [=The allow attribute of the iframe element=] section.
+
+  Rename it to "The allow attribute of the iframe and fencedframe element".
+
+  Rewrite the section to read:
+
+  &lt;iframe> and &lt;fencedframe> elements have an "allow" attribute, which contains an
```

Comment:
Same here, and elsewhere if applicable, regarding the element and attribute links.